src/css/App.css

```diff
@@ -282,6 +282,59 @@ color: #000000;
   margin-top: 0;
 }
 
+.auth{
+	background: #FFFFFF;
+	border: 1px solid #efefef;
+	border-radius:5px;
+	padding: 50px;
```

Comment:
@Etomovich, this padding seems to be too large in pixels.

Reply:
Cool let me work on the issue too